docs: Search Metadata Batch 04 (#14596)
hashicorp/consul: website/content/docs/connect/config-entries/terminating-gateway.mdx

---
layout: docs
page_title: Terminating Gateway: Configuration Entry Reference
description: >-

> The terminating gateway configuration entry kind defines proxy behavior to secure outgoing communication between the service mesh and external sources. Use the reference guide to learn about `""terminating-gateway""` config entry parameters and connecting service instances without Consul agents to your service mesh.

Reviewer: ```suggestion
The terminating gateway configuration entry kind defines proxy behavior to secure outgoing communication between the service mesh and external sources. Use the reference guide to learn about `""terminating-gateway""` config entry parameters and connecting to external services from your service mesh.
```
